CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+
+-   Automated unit and integrations tests are now executing (#5489)
```

Comment:
Please add a line under `### Changed` to indicate the move from Travis to Github Actions.

Reply:
Ah, yeah. That makes sense. Good idea.